Xamarin.Forms.Platform.Android/VisualElementPackager.cs

```diff
@@ -263,8 +264,11 @@ void RemoveChild(VisualElement view)
 			else
 			{
 				_childViews.Remove(renderer);
+
 				renderer.View.RemoveFromParent();
-				renderer.Dispose();
+
+				// Delay the dispose to let the animation finish
+				new Handler(Looper.MainLooper).Post(() => renderer.Dispose());
```

Comment:
I suspect that we'll now need to null check `renderer` and/or check if it's already disposed, since we're introducing a delay here.